infrastructure/stacks/terraform_management/provider.tf

```diff
@@ -17,9 +17,9 @@ provider "aws" {
 
   default_tags {
     tags = {
-      owner               = "UEC"
-      project             = "UEC DoS"
-      environment         = "Dev"
+      owner               = var.project_owner
+      project             = var.project
+      environment         = var.environment
       terraform-base-path = replace(path.cwd, "/^.*?(${local.terraform-git-repo}\\/)/", "$1")
```

Comment:
Think that you've already raised this in the test evidence. Currently this provides the complete local path if run via command line. Is this what we want?

Reply:
Yes I think i covered this in test evidence or ticket itself ; I think if always run from pipeline this will be okay and not change ; run from command line it could change which we would not want. I was for leaving it until the pipelinie is available and/or using it to help find resources not created via pipeline... one to discuss maybe

Reply:
Once the local terraform git repo variable is set correctly this problem goes away